crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -612,6 +616,25 @@ type Epoch {
 	"""
 	protocolConfigs: ProtocolConfigs
 	"""
+	Information about whether last epoch change used safe mode, which happens if the full epoch
+	change logic fails for some reason.
+	"""
+	safeMode: SafeMode
+	"""
+	The value of the `version` field of `0x5`, the `0x3::sui::SuiSystemState` object.  This
+	version changes whenever the fields contained in the system state object (held in a dynamic
+	field attached to `0x5`) change.
+	"""
+	systemStateVersion: BigInt
+	"""
+	Details of the system that are decided during genesis.
+	"""
+	systemParameters: SystemParameters
+	"""
+	Parameters related to subsiding staking rewards
+	"""
+	systemStakeSubsidy: StakeSubsidy
```

Comment:
While the stake subsidy does subside with time, I don't think that's why it's called that.  Also note that there are subsidy-related fields higher up in the `Epoch` table.  Now that this field is in the same table, it's worth clarifying that this parameter controls how the subsidy will work in this epoch (forward looking), while those other parameters describe how the subsidy was distributed (backward looking) for this epoch, and so only exist on past epochs.

```suggestion
	"""
	Parameters related to the subsidy that supplements staking rewards
	"""
	stakeSubsidy: StakeSubsidy
```